Shell/Preprocess.cpp

```diff
@@ -136,10 +152,8 @@ void Preprocess::preprocess(Problem& prb)
     GoalGuessing().apply(prb);
   }
 
-  // If there are interpreted operations
+  // interpreted normalizations are not prepeared for "special" terms, thus it must happen after clausification
   if (prb.hasInterpretedOperations() || env.signature->hasTermAlgebras()){
-    // Normalizer is needed, because the TheoryAxioms code assumes Normalized problem
-    InterpretedNormalizer().apply(prb);
```

Comment:
Have you considered whether it's a problem for the non-alasca strategies that we no longer do `InterpretedNormalizer` here? 

I mean, in the light of 
"// Normalizer is needed, because the TheoryAxioms code assumes Normalized problem"

Reply:
Hmmm good question. I can't quite remember when/why i removed this. 
I did loads of randomized testing, so I think potential issues should have showed up. Maybe we should also do a regression test checking whether the performance of the old portfolios matches the new one just to be sure i didn't end up skipping adding some theory axioms that were necessary by changing that. 

In order to be safe we could also do a guarded normalization with `if(!env.options->alasca())`... But I think it's probably not necessary. What do you think?

Reply:
Yes, that's the line of thinking I was hinting at! 

We had several issues in the past with the arithmetic code not doing what it should (performance-wise), because our pipeline was silently broken is such subtle ways like missing normalization steps at strange places during preprocessing. It's a good idea to do a performance regression. (And we should probably go through the pain of trying out both the old approach and alasca, to have them compared during spidering, before saying good buy to the old one completely.)